lib/services/ngsiService.js

```diff
@@ -129,7 +129,9 @@ function sendRegistrations(unregister, deviceData, callback) {
         } else if (response && body && response.statusCode === 200) {
             var errorField = getErrorField(body);
 
-            if (errorField) {
+			/* fixed an issue with the check of the errorField code. Now the error is not raised if the errorField code is equal to 200 (ReasonPhrase: OK) */
+
+            if (errorField && errorField.code!=200) {
```

Comment:
Better add a check for the existence of `errorField.code`, as the format for the error is quite free and unpredictable.